src/core/AutomationClip.cpp

```diff
@@ -863,7 +863,7 @@ void AutomationClip::saveSettings( QDomDocument & _doc, QDomElement & _this )
 
 
 
-void AutomationClip::loadSettings( const QDomElement & _this )
+void AutomationClip::loadSettings(const QDomElement & _this)
```

Comment:
Consider changing the `_this` name here, too
```suggestion
void AutomationClip::loadSettings(const QDomElement& _this)
```

Reply:
should i really? i'll pass on this for now.